installer/workflow-helper.sh

```diff
@@ -42,6 +42,8 @@ wget "$packet_base_url/ca.pem"
 mkdir -p /etc/docker/ /etc/docker/certs.d/ "/etc/docker/certs.d/$docker_registry"
 cp ca.pem "/etc/docker/certs.d/$docker_registry/ca.crt"
 
+# wait for docker to fully start
+sleep 3
```

Comment:
There is `until` loop right after it, which will retry if command fails. Is this sleep really needed?

Reply:
yes. in my dev environment docker login fails at least twice because docker has not fully started.

Reply:
But if there is a `until` loop, it should retry, so I think it's fine if it fails?

Reply:
sure i will remove the 1st sleep i added